core/controllers/admin.py

```diff
@@ -205,6 +293,54 @@ def _reload_collection(self, collection_id):
             raise Exception('Cannot reload a collection in production.')
 
 
+class AdminRoleHandler(base.BaseHandler):
+    """Handler for roles tab of admin page. Used to view and update roles."""
+
+    @require_super_admin
+    def get(self):
+        view_method = self.request.params['method']
+
+        if view_method == 'role':
+            role = self.request.params['role']
+            users_by_role = {
+                username: role
+                for username in user_services.get_usernames_by_role(role)
+            }
+            role_services.store_role_query(
+                self.user_id, feconf.VIEW_ROLE, method='role', role=role)
```

Comment:
'role', 'username' should be named constants. (VIEW_METHOD_ROLE, VIEW_METHOD_USERNAME) Use these constants everywhere you currently use the strings.